addons/event/models/event_event.py

```diff
@@ -130,7 +130,7 @@ def _lang_get(self):
         'res.partner', string='Organizer', tracking=True,
         default=lambda self: self.env.company.partner_id,
         check_company=True)
-    event_type_id = fields.Many2one('event.type', string='Template', ondelete='set null')
+    event_type_id = fields.Many2one('event.type', string='Template', ondelete='set null', help="Choose a template to auto-fill tickets, communications, descriptions and other fields.")
```

Comment:
Can probably go to new line.

Thanks for the changes. Greening already, "Menu Item" wording can be discussed later.

Reply:
Sure, will do that. Kindly refer to my recent comment above about the wording.
Thanks!